README.md

```diff
@@ -1,3 +1,3 @@
 Topic: Maybe I Know my place
 
-Application description- This web app is created as an inspiration from a very famous game a.k.a GeoGusser like the game where the player's are asked to predict the correct name of the place based on the place that ther are being show on there screens so is the same scene with these web app but with a lot of twist's and fun just to test there knowledge of the place if the player recognise the place then thay are asked to write a description about the place if they didn't recognise the place shown to them then they have the access to know about the place by the responce subitted by the previous user who already knew about the place in that way they have the access to even connect to the other players as well, Much more things will be added in the future.
\ No newline at end of file
+Application description- In this web page I will be displaying the list of images of the famous places all over the india ,so the user can see it and find out the places just by looking on the images.
```

Comment:
Consider merging the old and new descriptions to provide a more comprehensive overview of the application. For example: "In this web page, we display a list of images of famous places all over India. Users are challenged to identify these places based on the images. If a user recognizes a place, they are asked to write a description about it. If they don't recognize the place, they can learn about it from the responses submitted by previous users. This interactive format not only tests users' knowledge but also allows them to connect with other players. We plan to add more features in the future." [medium]